README.md

```diff
@@ -104,14 +99,19 @@ See **Installation Instructions** for each available [release](https://github.co
 > | [v2022-12-14](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2022-12-14) | [1.35.4](https://github.com/input-output-hk/cardano-node/releases/tag/1.35.4) |
 > | [v2022-10-06](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2022-10-06) | [1.35.3](https://github.com/input-output-hk/cardano-node/releases/tag/1.35.3) |
 
-## How to build from sources
+### Building from source
 
 See [Building](https://cardano-foundation.github.io/cardano-wallet/developers/Building)
 
-## How to test
+### Testing
 
 See [Testing](https://cardano-foundation.github.io/cardano-wallet/contributing/Testing)
 
+## History
+
+The `cardano-wallet` repository has been introduced for the [Shelley phase](https://roadmap.cardano.org/) of the Cardano blockchain.
+Previously, during the Byron phase, the wallet was part of the [cardano-sl](https://github.com/input-output-hk/cardano-sl) repository. (This is useful to know — sometimes, the ghosts of the past haunt us in the form of obscure bugs.)
```

Comment:
Would suggest to remove the `,`. When reading an em-dash, the reader will usually pause, so it feels a bit unnatural to pause again only one word later.

Additionally, I suggest that we add "come back to" before "haunt us", since "come back to haunt us" is a commonly used idiom that's very natural in this context.

```suggestion
Previously, during the Byron phase, the wallet was part of the [cardano-sl](https://github.com/input-output-hk/cardano-sl) repository. (This is useful to know — sometimes the ghosts of the past come back to haunt us in the form of obscure bugs.)
```

Reply:
👍